test/python/test_rticonnextdds_performance.py

```diff
@@ -0,0 +1,100 @@
+###############################################################################
+# (c) 2020 Copyright, Real-Time Innovations.  All rights reserved.            #
+# No duplications, whole or partial, manual or electronic, may be made        #
+# without express written permission.  Any such copies, or revisions thereof, #
+# must display this notice unaltered.                                         #
+# This code contains trade secrets of Real-Time Innovations, Inc.             #
+###############################################################################
+
+import pytest,sys,os
+sys.path.append(os.path.dirname(os.path.realpath(__file__))+ "/../../")
+import rticonnextdds_connector as rti
+from test_utils import *
+
+# These tests currently take around 25 minutes to run with 100 repeats.
+# we do. They were added to help verify performance of anothe product pyrti API.
+# If you want to run them, remove this decorator.
+# Maybe once CON-42 is implemented we can run them by default.
+@pytest.mark.skip(reason="Takes too long to run these tests, remove this line and run manually")
+class TestPerformance:
+    """
+    This class tests the performance of Connector
+    """
+
+    # How many times to repeat each test
+    @pytest.fixture(scope="class")
+    def num_iterations(self):
+        return 100
+
+    # Test how long it takes to set a sequence of 600000 elements.
+    # There are 2 options available:
+    # for i in range(0, 600000):
+    #   output.instance['myOctSeq[i]'] = 1
+    # or
+    # output.instance['myOctSeq'] = [1, 2, 3, 4, 5, ....]
+    # Test both approaches (expected to be slow until we implement CON-42)
+    def test_set_sequence(self, one_use_connector, num_iterations):
+        # Get the input and output which communicate using the performance test type
+        the_input = one_use_connector.get_input("MySubscriber::PerformanceTestReader")
+        the_output = one_use_connector.get_output("MyPublisher::PerformanceTestWriter")
+
+        # Wait for discovery between the entities
+        the_input.wait_for_publications(5000)
+        the_output.wait_for_subscriptions(5000)
+
+        # Approach 1.
+        # Set each element of the sequence separately
+        total_time = 0
+        for i in range(0, num_iterations):
+            start_time = time.time()
+            for i in range (0, 600000):
+                the_output.instance['myOctSeq[%d]' % (i)] = 2
+            total_time += (time.time() - start_time)
+        average_time = total_time / num_iterations
+        print("Average time setting element-by-element: " + str(average_time))
+        # Average time setting element-by-element: 2.9646800351142883
+
+
+        # Approach 2.
```

Comment:
Do we want to separate this somehow so if needed we could run one test or the other?